Userland/Games/Solitaire/Game.h

```diff
@@ -37,10 +37,12 @@ class Game final : public GUI::Frame {
 
     Mode mode() const { return m_mode; }
     void setup(Mode);
+    void undo_perform();
```

Comment:
I think the more common way to express this in the codebase is `perform_<action>` or `do_<action>`, perhaps use one of those? (I suggest `perform_undo`)